starter/nextjs/next.config.js

```diff
@@ -0,0 +1,18 @@
+/** @type {import('next').NextConfig} */
+
+const url = new URL(process.env.NEXT_PUBLIC_DOTCMS_HOST);
+
+const nextConfig = {
+    images: {
+        remotePatterns: [
```

Comment:
Do you think it might be useful to add some comment on why this is done? Or is anyone who uses Next clear on why?